gammapy/data/event_list.py

```diff
@@ -132,7 +132,14 @@ def altaz(self):
 
         lon, lat = self['AZ'], self['ALT']
         return SkyCoord(lon, lat, unit='deg', frame=altaz_frame)
-
+    
+    @property
+    def offset(self):
```

Comment:
The orientation of DETX / DETY isn't well-defined yet (axes aligned with ALT/AZ or RA/DEC).

For HESS the pointing position in RADEC is fixed during the whole run.
The separation of this pointing RADEC wrt. the event RADEC should be the same as the offset computed with DETX / DETY.
If this is not the case, there's either a bug in Gammapy or the event list.
Can you try to find out what's going on here or should I investigate?